### PR TITLE
Add GA methods and goals

### PR DESCRIPTION
**Pull request by OverVlad**

### CODE REVIEW

**Comment by OverVlad** [comment]
[Set up GA goals](https://trello.com/c/h2d8hRcf/345-set-up-ga-goals)

File: app/concepts/stepper/assets/stepper.js [hunk 1]
```diff
@@ -112,7 +120,7 @@
     Stepper.prototype.finish = function() {
         _chatlio.show({expanded: true});
 
-        trackEventInGoogleAnalytics('finish action', this.getTimestamp());
+        trackEventInGoogleAnalytics('finish action');
```
Comment (solopiy) [inline app/concepts/stepper/assets/stepper.js]: Don't we need to track time when the user actions are completed? Or maybe GA doing it automatically

File: app/concepts/stepper/assets/stepper.js [hunk 2]
```diff
@@ -86,7 +94,7 @@
 
     Stepper.prototype.changeStep = function(newPath) {
         this.hideSection(this.$currentSection)
-            .showSection(this.getSection(newPath));
```
Comment (solopiy) [inline app/concepts/stepper/assets/stepper.js]: Why do we need to remove this empty space?

File: app/concepts/stepper/assets/stepper.js (hunk 1 above)
Comment (OverVlad) [inline app/concepts/stepper/assets/stepper.js]: @solopiy GA can't track time anyone. It considers that goals complete if user, for example, use form more than 3 or 5 seconds

**Comment by OverVlad** [comment]
@solopiy you use .dev GA account? You can check goals in section Real Time > Conversion

**Pull request merged**